src/pages/components/CertificateForm.js

```diff
@@ -1,16 +1,61 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
 
 import { Grid } from '@digicatapult/ui-component-library'
 
+import CertificateInputFields from './CertificateInputFields'
+import CertificateFormHeader from './CertificateFormHeader'
+import CertificateActionsButtons from './CertificateActionsButtons'
+
 export default function CertificateForm() {
+  const [sdVal, setSdVal] = useState('2024-01-01')
+  const [stVal, setStVal] = useState('00:00')
+  const [edVal, setEdVal] = useState('2024-01-01')
+  const [etVal, setEtVal] = useState('23:55')
+  const [enVal, setEnVal] = useState('')
+  const [szVal, setSzVal] = useState('')
+
+  const handleSdChgeVal = (e) => setSdVal(e.target.value)
+  const handleStChgeVal = (e) => setStVal(e.target.value)
+  const handleEdChgeVal = (e) => setEdVal(e.target.value)
+  const handleEtChgeVal = (e) => setEtVal(e.target.value)
+  const handleEnChgeVal = (e) => setEnVal(e.target.value)
+  const handleSzChgeVal = (e) => setSzVal(e.target.value)
+
```

Comment:
nice, I do like this implementation as it's in it's simplest form where we could easily add a story in the backlog for improving

Maybe something I would look into is abstracting the below
```js

  const handleSdChgeVal = (e) => setSdVal(e.target.value)
  const handleStChgeVal = (e) => setStVal(e.target.value)
  const handleEdChgeVal = (e) => setEdVal(e.target.value)
  const handleEtChgeVal = (e) => setEtVal(e.target.value)
  const handleEnChgeVal = (e) => setEnVal(e.target.value)
  const handleSzChgeVal = (e) => setSzVal(e.target.value)
  
  // into something like
  const [val, setVal ] = React.useState({ ed: '2020-10-10', et: '20:20', en: '', sz: '' }) // can also be just an empty object e.g. React.useState({})
  const handleChange (e, key) => setVal({ ...val, [key]: e.target.value }) 
```